stochastic/continuous/mixed_poisson.py

```diff
@@ -0,0 +1,92 @@
+"""Mixed poisson processes."""
+
+from stochastic.base import Checks
+from stochastic.continuous import PoissonProcess
```

Comment:
Change this to 
`from stochastic.continuous.poisson import PoissonProcess`

This should eliminate the circular import that appears to be causing the tests to fail.